ci/publish

```diff
@@ -45,7 +45,8 @@ if [ "${TRAVIS_TAG:-}" == "" ]; then
     #       jupyterhub/binderhub@123abcd which will be understood as a reference
     #       by GitHub.
     PR_OR_HASH=$(git log -1 --pretty=%B | head -n1 | sed 's/^.*\(#[0-9]*\).*/\1/' | sed 's/^\([0-9a-f]*\)-.*/@\1/')
```

Comment:
```suggestion
    PR_OR_HASH=$(git log -1 --pretty=%h-%B | head -n1 | sed 's/^.*\(#[0-9]*\).*/\1/' | sed 's/^\([0-9a-f]*\)-.*/@\1/')
```
is this what you meant as a fix to the `PR_OR_HASH` @consideRatio ?

Reply:
Yepp @betatim thank you!! This makes the `@123abcd` hash be created successfully when the commit title doesn't container `#123` as only the GitHub merge commits can be assumed to do.